docs/src/docs/demos/core/Hr/hr.demos.tsx

```diff
@@ -76,3 +76,94 @@ export function HrSizesDemo() {
     </CodeDemo>
   );
 }
+
+const subHeaderCode = `import React from 'react';
+import { Hr } from '@mantine/core';
+function Demo() {
+  return (
+    <>
+      <Hr subHeader="Test" />
+      <Hr
+        subHeader="Test"
+        subHeaderProps={{ style: { textAlign: 'center' }, color: 'red' }}
+        style={{ marginTop: 10 }}
+      />
+      <Hr
+        subHeader="Test"
+        subHeaderProps={{ style: { textAlign: 'end', fontFamily: 'cursive' } }}
+        style={{ marginTop: 10 }}
+      />
+      <Hr 
+        subHeader="Test" 
+        subHeaderProps={{ style: { fontSize: 20 } }} 
+        style={{ marginTop: 15 }} 
+      />
+    </>
+  );
+}`;
+
+export function HrSubHeaderDemo() {
+  return (
+    <CodeDemo code={subHeaderCode} language="tsx">
+      <Hr subHeader="Test" />
+      <Hr
+        subHeader="Test"
+        subHeaderProps={{ style: { textAlign: 'center' }, color: 'red' }}
+        style={{ marginTop: 10 }}
+      />
+      <Hr
+        subHeader="Test"
+        subHeaderProps={{ style: { textAlign: 'end', fontFamily: 'cursive' } }}
+        style={{ marginTop: 10 }}
+      />
+      <Hr subHeader="Test" subHeaderProps={{ style: { fontSize: 20 } }} style={{ marginTop: 15 }} />
+    </CodeDemo>
+  );
+}
+
+const orientationCode = `import React from 'react';
+import { Hr } from '@mantine/core';
+
+function Demo() {
+  return (
+    <>
+      <Hr orientation="horizontal" />
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          justifyContent: 'space-evenly',
+          marginTop: 15,
+        }}
+      >
+        <Badge>Light</Badge>
+        <Hr orientation="vertical" />
+        <Badge variant="outline">Outline</Badge>
```

Comment:
It's Hr docs, not Badge, remove variants from badge

Reply:
Text would work? @rtivital

Reply:
You can leave badges but remove variants from them

Reply:
Okay